app/components/SwapDetails.tsx

```diff
@@ -29,14 +27,14 @@ export default function SwapDetails(props: SwapDetailsProps) {
           <Tag p={3} variantColor="cyan" width="45%">
             <TagLabel>
               {/* TODO: hack to remove trailing zeros */}
-              {parseFloat(insertDecimals(properties.buy_quantity, 8))}{' '}
-              {getAssetName(properties.buy_asset)}
+              {/* {parseFloat(insertDecimals(properties.buy_quantity, 8))}{' '} */}
+              {/* {getAssetName(properties.buy_asset)} */}
```

Comment:
Is this still needed or can we remove it?

Reply:
The code is currently not used, but might be in the future again. It is currently causing linting problems, which is annoying. Since I don't know the code base that well I opted for commenting it instead of just removing it. I would potentially use/remove it once I (re-)implement the swap execution.